### PR TITLE
ESlint fixes and flowification of virtualization JS files

### PR DESCRIPTION
**Pull request by cbosdo**
## What does this PR change?

This is just code cleanup in the end.

## GUI diff

No difference.

- [X] **DONE**

## Documentation
- No documentation needed: code cleanup needs no doc

- [X] **DONE**

## Test coverage
- No tests: code cleanup, just need existing tests to pass again

- [X] **DONE**

## Links

- [X] **DONE**

### CODE REVIEW

**Comment by cbosdo** [comment]
remains to be decided what to do with the `'use strict'`: I left them only in the renderer files since those are the top-most ones, but if you say it's OK to remove them everywhere then I can fix these last errors.

**Comment by cbosdo** [comment]
gitarro no changelog needed !!!

**Comment by cbosdo** [comment]
gitarro rerun changelog_test !!!

File: web/html/src/manager/virtualization/guests/list/guests-list.js [hunk 1]
```diff
@@ -1,406 +1,441 @@
...
+  pageUnloading: boolean,
+  websocketErr: boolean,
+};
 
-    this.handleSelectItems = (items) => {
-      this.setState({
-        selectedItems: items
-      });
+class GuestsList extends React.Component<Props, State> {
+  static sortByUpdate(aRaw: Object, bRaw: Object, columnKey: string, sortDirection: number) {
```
Comment (lneves12) [inline web/html/src/manager/virtualization/guests/list/guests-list.js]: I wonder if these static functions wouldn't be better inside some kind of local utils file, so the component GuestList gets a bit less cumbersome. It's just a remark for future :)

**Pull request merged**